### PR TITLE
Slider control

### PR DESCRIPTION
**Pull request by kerpecheva**

### CODE REVIEW

File: controls/slider/styling/labels-styling.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Labels Styling
+page_title: .NET MAUI Slider Documentation - Labels Styling
+description: Review the styling options that the Telerik UI for .NET MAUI Slider control provides for its labels.
```
Comment (ikoevska) [inline controls/slider/styling/labels-styling.md:4]:
```suggestion
description: Learn how to style the labels of the Telerik UI for .NET MAUI Slider. Set the colors and fonts, or use the style selector for a more complex styling approach.
```

File: controls/slider/styling/ticks-styling.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+title: Ticks Styling
+page_title: .NET MAUI Slider Documentation - Ticks Styling
+description: Review the styling options that the Telerik UI for .NET MAUI Slider control provides for its ticks.
```
Comment (ikoevska) [inline controls/slider/styling/ticks-styling.md:4]:
```suggestion
description: Learn how to style the ticks of the Telerik UI for .NET MAUI Slider. Set the size and style ticks within and outside of the range track differently.
```

File: controls/slider/labels.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+<snippet id='slider-labels-labeltemplate-xaml' />
+
+Here is the result:
+
+![Telerik Slider for .NET MAUI Label Template](images/slider-labels-template.png)
+
+## See Also
+
+- [Ticks]({% slug slider-ticks%})
+- [Labels Styling]({% slug slider-labels-styling%})
```
Comment (ikoevska) [inline controls/slider/labels.md:73]:
```suggestion
- [Tooltips]({% slug slider-tooltips %})
- [Labels Styling]({% slug slider-labels-styling%})
```

File: controls/slider/labels.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+* `StringFormat`(`string`)&mdash;Defines a custom string format for the labels and the tooltips of the Slider.
+* `StringConverter`(`Telerik.Maui.IStringConverter`)&mdash;Specifies a custom string converter that can be used to define the content of a label or a tooltip for a given slider value.
+
+Here is a quick example with a custom string converter:
+
+**1.** Add the following sample Dictionary String Converter to your resources&mdash;in short, it replaces the values of the range with meaningful names in order to make the range more readable:
+
+<snippet id='slider-labels-stringconverter-dictionary' />
+
+**2.** Define the Slider with the converter applied:
```
Comment (ikoevska) [inline controls/slider/labels.md:42]:
```suggestion
**2.** Apply the converter to the Slider:
```

File: controls/slider/labels.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+## Label Template
+
+You can customize what the Slider labels render through the `LabelTemplate` property.
+
+* `LabelTemplate`(`DataTemplate`)&mdash;Defines the template of the Slider labels.
+
+Check below a sample `LabelTemplate` example:
+
+**1.** First define the custom DataTemplate:
```
Comment (ikoevska) [inline controls/slider/labels.md:58]:
```suggestion
**1.** Add the custom `DataTemplate` to your page resources:
```

**Review by ikoevska — COMMENTED** [review]
Don't be alarmed by the number of comments. All in all, that's a great PR with a lot of content. For the most part, my comments are suggestions that aim to apply consistency where it makes sense. I also took the liberty of suggesting more detailed description fields, where it made sense. 

Outside of inline comments, there are two things that I'd like to discuss:
* There's no article for styling tooltips. Is this by design? Or is it covered by the Tooltips article?
* The article is called "value thumb", but in fact, in most places, it's referred to as just "thumb". I would suggest using one or the other for consistency.

File: controls/slider/overview.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+ The Telerik UI for .NET MAUI Slider represents a slider control that displays a value in a given min-max range. The end user can change the values by dragging the thumb. Slider supports ticks, labels and tooltips to help users quickly identify the min-max range and what the value is as well as easily modify them if needed. 
+
+![.NET MAUI Slider Overview](images/slider-overview.png)
+
+## Key Features
+
+* [Value Thumb]({%slug slider-value-thumb%})&mdash;The slider's value thumb is drawn corresponding to its `Value` property. Users can modify the `Value` by dragging the thumb along the backtrack.
+* [Range Track]({%slug slider-range-track%})&mdash;The slider provides customizable range track for better representation of the selected value. You can modify the range track initial position through the `OriginValue` property.
+* [Ticks]({%slug slider-ticks%})&mdash;Add ticks to the slider's track in order to enable users to easily identify the min-max range.
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply controls/slider/overview.md:21]: Agree with that note, I missed that with this bullet list

File: controls/slider/overview.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+* [Labels]({%slug slider-labels%})&mdash;Display labels along the track for clarity of what the underlying min-max range is.
+* [Tooltip]({%slug slider-tooltips%})&mdash;Slider can show customizable tooltip to help the users choose the desired value.
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply controls/slider/overview.md:23]: Actually it is only one tooltip - it is shown when the thumb is dragged.

File: controls/slider/overview.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+* Templates&mdash;Further customize the look of the Slider's labels, ticks and tooltip by using the exposed templates.
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply controls/slider/overview.md:24]: It is only one tooltip in the control

File: controls/slider/value-thumb.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+* `Value`(`double`)&mdash;Specifies the selected value of the slider, corresponds to the thumb position across the backtrack.
+
+Check a quick example on how to define `Value` property:
+
+<snippet id='slider-getting-started-xaml' />
+
+## Range Track
+
+In addition to the thumb presenting the slider's `Value`, you can utilize the ran
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply controls/slider/value-thumb.md]: Already moved to a separate article

File: controls/slider/overview.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (ikoevska) [reply controls/slider/overview.md:23]: I have amended the suggestion :) It probably goes without saying, but, please, apply the singular consistently throughout the set.

File: controls/slider/overview.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (ikoevska) [reply controls/slider/overview.md:24]: I have updated the suggestion as needed :)

**Comment by ikoevska** [comment]
Let me know when you want me to take a second look :)

File: controls/slider/backtrack-configuration.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
+---
+title: Backtrack Configuration
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply controls/slider/backtrack-configuration.md:2]: Agree

**Comment by kerpecheva** [comment]
> Don't be alarmed by the number of comments. All in all, that's a great PR with a lot of content. For the most part, my comments are suggestions that aim to apply consistency where it makes sense. I also took the liberty of suggesting more detailed description fields, where it made sense.
> 
> Outside of inline comments, there are two things that I'd like to discuss:
> 
> * There's no article for styling tooltips. Is this by design? Or is it covered by the Tooltips article?

there are no styling options, as the tooltip can be fully modified with the TooltipControlTemplate

> * The article is called "value thumb", but in fact, in most places, it's referred to as just "thumb". I would suggest using one or the other for consistency.

Named it like this for consistency with the RangeSlider, where it is Range Thumb, to emphasize the thumb actually sets the Value

File: controls/slider/events.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+    if (e.Value < 20)
+    {
+        e.Value = slider.Value;
+    }
+}
+```
+
+## See Also
+
+- [Value Thumb]({%slug slider-value-thumb%})
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply controls/slider/events.md:44]: Agree

File: controls/slider/labels.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+
+Check an example on how labels can be configured:
+
+<snippet id='slider-labels-settings' />
+
+![Telerik Slider for .NET MAUI Labels](images/slider-labels-settings.png)
+
+## Labels Formatting
+
+Through the formatting properties you can easily modify only the labels text by applying a custom format.
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply controls/slider/labels.md:31]: I'd use the updated sentence without programmatically - as in general we use "programmatically" for code-behind, while the format properties can also be set with XAML.

File: controls/slider/labels.md [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+
+<snippet id='slider-labels-stringconverter' />
+
+Check the result below:
+
+![Telerik Slider for .NET MAUI Labels StringConverter](images/slider-labels-stringconverter.png)
+
+## Label Template
+
+You can customize what the Slider labels render through the `LabelTemplate` property.
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply controls/slider/labels.md:52]: this is a property of the Slider, so I don't think the "across one or more Sliders in your app" is needed here.

File: controls/slider/tooltips.md [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+# Tooltips
+
+Slider for .NET MAUI can display a tooltip to indicate the currently selected value to the end-user. The toolip is shown as soon as the user starts dragging the thumb.
+
+Check below the customization options for the Slider tooltip:
+
+* `TooltipStringFormat`(`string`)&mdash;Defines a custom string format that is used for displaying the content of the tooltip.
+* `TooltipTemplate`(`DataTemplate`)&mdash;Sets a template or template selector that defines the content of the tooltip.
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply controls/slider/tooltips.md:16]: indeed, it's TooltipContentTemplate, fixed

File: controls/slider/tooltips.md [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+* `TooltipControlTemplate`(`ControlTemplate`)&mdash;Sets the control template of the tooltip that defines its overall appearance.
+
+## TooltipTemplate Example
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply controls/slider/tooltips.md:19]: fixed

File: controls/slider/value-thumb.md [hunk 16]
```diff
@@ -0,0 +1,52 @@
...
+
+## Drag Mode
+
+The Slider supports different dragging options&mdash;only the thumb or free. In addition, you can completely disable the drag. 
+
+You can control the available dragging options through the `DragMode` property of the Slider:
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply controls/slider/value-thumb.md:23]: fixed

File: controls/slider/value-thumb.md [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+
+* `DragMode`(`Telerik.Maui.Controls.Slider.SliderDragMode`)&mdash;Defines the possible ways to interact with the Slider and change its `Value`. You can choose from:
+    * `Disabled`&mdash;drag is not enabled.
+    * `Thumb`&mdash;the thumb can be dragged.
+    * `Free`&mdash;touching anywhere (the thumb, or the range track, or the backtrack) will lead to the `Value` being changed.
+ 
+Check below some quick examples:
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply controls/slider/value-thumb.md:30]: agree

File: controls/slider/visual-structure.md [hunk 18]
```diff
@@ -0,0 +1,29 @@
+---
+title: Visual Structure
+page_title: .NET MAUI Slider Documentation - Visual Structure
+description: Learn what visual elements are displayed by the Telerik UI for .NET MAUI Slider, and see how these elements build the visual structure of the control.
+position: 2
+slug: slider-visual-structure
+---
+
+# .NET MAUI Silder Visual Structure
```
Comment (ikoevska) [inline controls/slider/visual-structure.md:9]:
```suggestion
# .NET MAUI Slider Visual Structure
```

File: controls/slider/visual-structure.md [hunk 19]
```diff
@@ -0,0 +1,29 @@
...
+
+![.NET MAUI Slider Visual Structure](images/slider-visual-structure.png "Visual elements of Slider control")
+
+## Displayed Elements
+
+- **Backtrack**&mdash;The part of the slider across which the value thumb slides.
+- **Thumb**&mdash;A draggable handle which represents the slider value.
+- **Range Track**&mdash;The part of the backtrack from its start along to the value thumb.
+- **Ticks** and **Labels**&mdash;Visual indicators of the underlying min-max range.
+- **Tooltips**&mdash; Visual indicators of the current value selected with the thumb.
```
Comment (ikoevska) [inline controls/slider/visual-structure.md:23]:
```suggestion
- **Tooltip**&mdash; A visual indicator of the current value selected with the thumb.
```

File: controls/slider/value-thumb.md [hunk 20]
```diff
@@ -0,0 +1,46 @@
...
+<snippet id='slider-getting-started-xaml' />
+
+## Drag Mode
+
+The Slider's `DragMode` property lets you configure one of three dragging options&mdash;only the thumb, free, or disabled.
+
+* `DragMode`(`Telerik.Maui.Controls.Slider.SliderDragMode`)&mdash;Defines the possible ways to interact with the Slider and change its `Value`. You can choose from:
+    * `Thumb`&mdash;The thumb moves along the backtrack only by dragging.
+    * `Free`&mdash;The thumb's position (and the `Value` property) changes via dragging or touching anywhere on the range track or backtrack.
+    * `Disabled`&mdashThe thumb's position is fixed on the backtrack and dragging is disabled.
```
Comment (ikoevska) [inline controls/slider/value-thumb.md:26]:
```suggestion
    * `Disabled`&mdash;The thumb's position is fixed on the backtrack and dragging is disabled.
```

File: controls/slider/backtrack.md [hunk 21]
```diff
@@ -0,0 +1,45 @@
...
+title: Backtrack
+page_title: .NET MAUI Slider Documentation - Backtrack
+description: Learn how to configure the backtrack of the Telerik UI Slider for .NET MAUI provides - set the min and max values and choose if the backtrack should extend beyond them.
+position: 5
+slug: slider-backtrack
+---
+
+# .NET MAUI Slider Backtrack
+
+The thumb runs along the backtrack of the slider. The backtrack represents all the values your end users can choose from.
```
Comment (ikoevska) [inline controls/slider/backtrack.md:11]:
```suggestion
The thumb runs along the backtrack of the Slider. The backtrack represents all the values your end users can choose from.
```

File: controls/slider/backtrack.md [hunk 22]
```diff
@@ -0,0 +1,45 @@
...
+
+## Minimum and Maximum
+
+You need to configure a `Minimum` value and a `Maximum` value for the Slider. These values define the limits of the backtrack.
+
+* `Minimum`(`double`)&mdash;Defines the minimum value of the slider.
```
Comment (ikoevska) [inline controls/slider/backtrack.md:17]:
```suggestion
* `Minimum`(`double`)&mdash;Defines the minimum value of the Slider.
```

File: controls/slider/backtrack.md [hunk 23]
```diff
@@ -0,0 +1,45 @@
...
+* `Maximum`(`double`)&mdash;Defines the maximum value of the slider.
```
Comment (ikoevska) [inline controls/slider/backtrack.md:18]:
```suggestion
* `Maximum`(`double`)&mdash;Defines the maximum value of the Slider.
```

File: controls/slider/backtrack.md [hunk 24]
```diff
@@ -0,0 +1,45 @@
...
+                    LabelsPlacement="End"
+                    BackTrackExtent="50"/>
+```
+
+![Telerik Slider for .NET MAUI Backtrack Extent](images/slider-backtrack-extent.png)
+
+## See Also
+
+* [Value Thumb]({% slug slider-value-thumb%})
+* [Range Track)({%slug slider-range-track%})
```
Comment (ikoevska) [inline controls/slider/backtrack.md:44]:
```suggestion
* [Range Track]({%slug slider-range-track%})
```

File: controls/slider/tooltips.md [hunk 25]
```diff
@@ -0,0 +1,53 @@
...
+
+The Slider tooltip provides the following customization options:
+
+* `TooltipStringFormat`(`string`)&mdash;Defines a custom string format for the tooltip.
+* `TooltipContentTemplate`(`DataTemplate`)&mdash;Sets a template or template selector that defines the content of the tooltip.
+* `TooltipControlTemplate`(`ControlTemplate`)&mdash;Sets the control template of the tooltip that defines its overall appearance.
+
+## TooltipContentTemplate Example
+
+Check below a quick example with setting `TooltipStringFormat` and `TooltipContentTemplate` properties:
```
Comment (ikoevska) [inline controls/slider/tooltips.md:21]:
```suggestion
Check below a quick example with setting the `TooltipStringFormat` and `TooltipContentTemplate` properties:
```

File: controls/slider/styling/ticks-styling.md [hunk 26]
```diff
@@ -0,0 +1,62 @@
...
+
+**2.** Apply the custom styles to the Slider:
+
+<snippet id='slider-ticksttyling-xaml' />
+
+![Telerik Slider for .NET MAUI Ticks Styling](images/slider-ticks-styling.png)
+
+## TickStyleSelector Example
+
+The following example demonstrates how to use the `TicksStyleSelector` set two different styles for the major and minor ticks:
```
Comment (ikoevska) [inline controls/slider/styling/ticks-styling.md:37]:
```suggestion
The following example demonstrates how to use the `TicksStyleSelector` to set two different styles for the major and minor ticks:
```

File: controls/slider/events.md [hunk 27]
```diff
@@ -0,0 +1,45 @@
...
+- [Range Track)({%slug slider-range-track%})
```
Comment (ikoevska) [inline controls/slider/events.md:45]:
```suggestion
- [Range Track]({%slug slider-range-track%})
```

**Review by ikoevska — APPROVED** [review]
🚀

**Pull request merged**